pyuploadcare/api_resources.py

```diff
@@ -231,12 +231,28 @@ def store(self):
         """
         self._info_cache = rest_request('PUT', self._api_storage_uri)
 
-    def copy(self, effects=None, target=None):
+    def copy(self, effects=None, target=None, make_public=None, store=None, pattern=None):
         """Creates File copy
```

Comment:
I suggest to reformat function definition like this

def copy(self, effects=None, store=None, target=None, make_public=None, pattern=None):

And to change the order of description of params in docstring, in accordance with function definition. Of course, description of ‘store’ is needed.